charts/super-agent/values.yaml

```diff
@@ -0,0 +1,181 @@
+      remediation:
+        retries: 3  # Retries is the number of retries that should be attempted on failures before givin up. An uninstall is performed between each attempt.
+        strategy: rollback  # rollback or uninstall.
+
+# -- Values for the Flux chat. Ref.: https://github.com/fluxcd-community/helm-charts/blob/flux2-2.10.2/charts/flux2/values.yaml
+# @default -- See `values.yaml`
+flux2:
+  # -- Enable or disable FluxCD completely. New Relic' Super Agent need Flux to work, but the user can use an already
+  # existing Flux deployment. With that use case, the use can disable Flux and use this chart to only install the CRs
+  # to deploy the Super Agent.
+  enabled: true
+
+  # -- The installation of the CRDs is managed by the chart itself.
+  installCRDs: true
+  # -- This is the domain name of the cluster.
+  clusterDomain: cluster.local
+
+  # -- Source controller provides a way to fetch artifacts to the rest of controllers. The source API (which reference
+  # [can be read here](https://fluxcd.io/flux/components/source/api/v1/)) is used by admins and various automated
+  # operators to offload the Git, OCO, and Helm repositories management.
+  # @default -- Enabled
+  sourceController:
+    create: true
+    annotations:
+      prometheus.io/port: "8080"
+      prometheus.io/scrape: "false"  # TODO: Should we create a FluxCD entity to monitor flux itself!?
+
+  # -- Helm controller is a Kubernetes operator that allows to declaratively manage Helm chart releases with Kubernetes
+  # manifests. The Helm release is defined in a CR ([Custom Resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#custom-resources))
+  # named `HelmRelease` that the operator will reconcile on the apply, edit, or deletion of a `HelmRelease` resource.
+  #
+  # New Relic' Super Agent will use this controller by creating `HelmRelease` CRs based in the configuration stored on
+  # OpAmp.
+  # @default -- Enabled
+  helmController:
+    create: true
+    annotations:
+      prometheus.io/port: "8080"
+      prometheus.io/scrape: "false"  # TODO: Should we create a FluxCD entity to monitor flux itself!?
+
+  # -- This controller is exactly the same as the Helm controller (refer to it) but assembling manifests with
+  # Kustomize instead of using templating systems like Helm.
+  # 
+  # New Relic' Super Agent will use this controller by creating `Kustomization` CRs based in the configuration stored
+  # on OpAmp.
+  # @default -- Enabled
+  kustomizeController:
+    create: true
+    annotations:
+      prometheus.io/port: "8080"
+      prometheus.io/scrape: "false"  # TODO: Should we create a FluxCD entity to monitor flux itself!?
+
+  # -- The image automation controller updates YAML files based on the latest images scanned by image reflector
+  # controller, and commits the changes to a given Git repository.
+  #
+  # From New Relic, all releases are managed via OpAmp and there is no need to touch user's repositories.
```

Comment:
I have to set `create` below to false. From a user perspective, it is better to have the rationale for why it is not being created.

I would prefer to rephrase it rather than remove it.

The other alternative would be to create a custom chart for Flux which is a possibility that I did not consider as it would add way more maintainability effort.

Reply:
We could state that SA only use the helm/source controller to deploy sub agents and the other Flux components are disable by default. , I would put all disable configs together under the same comment

Reply:
Fair. Done on [#7d0f2f0](https://github.com/newrelic/helm-charts/pull/1175/commits/7d0f2f0556057c63203ac9b721dad8ca91b9b3b6)